environment.yml

```diff
@@ -22,6 +22,8 @@ dependencies:
   - flake8
   - boto3-stubs-essential
   - pip:
+    - git+ssh://git@github.com/MetOffice/edr_server@CAP-340_add_collection_deserialisation
```

Comment:
This repo is public now, so you can switch the URL to the `https` version and it should fix the issue with the build not running properly due to permissions